WooCommerce/Classes/POS/Presentation/PointOfSaleExitPosAlertView.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+
+struct PointOfSaleExitPosAlertView: View {
+    @Environment(\.dismiss) private var dismiss
+    @ObservedObject private var viewModel: PointOfSaleDashboardViewModel
+
+    init(viewModel: PointOfSaleDashboardViewModel) {
+        self.viewModel = viewModel
+    }
```

Comment:
It would be nice if this just had the `isPresented` binding, not the whole view model

Reply:
Done in 358d15c

Reply:
Yep, good point, tnx!